index.html

```diff
@@ -0,0 +1,168 @@
+                </h1>
+                <h4 class="supporting-text">
+                    Software Engineer
+                </h4>
+            </div>
+            <div class="intro-text">
+                <p>I can help you build a product , feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don't hestiate to contact me.
+                </p>
+            </div>
+            <hr>
+            <div class="about-services">
+                <div class="service-list">
+                    <div class="title">
+                        <h6>Language</h6>
+                    </div>
+                    <ul>
+                        <li><a href="#javascript">Javascript</a></li>
+                        <li><a href="#ruby">Ruby</a></li>
+                        <li><a href="#html">Html</a></li>
+                        <li><a href="#css">Css</a></li>
+                    </ul>
+                </div>
+                <div class="service-list">
+                    <div class="title">
+                        <h6>Framework</h6>
+                    </div>
+                    <ul>
+                        <li><a href="#bootstrap">Boostrap</a></li>
+                        <li><a href="#ruby">Ruby on Rails</a></li>
+                        <li><a href="#respec">RSPec</a></li>
+                        <li><a href="#capy">CapyBara</a></li>
+                        <li><a href="#sele">Selenium</a></li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="about-services2">
+                <div class="service-list">
+                    <div class="title">
+                        <h6>Skills</h6>
+                    </div>
+                    <ul>
+                        <li><a href="#code">Codekit</a></li>
+                        <li><a href="#git">Github</a></li>
+                        <li><a href="#codepen">Codepen</a></li>
+                        <li><a href="#gitlab">Gitlab</a></li>
+                        <li><a href="#terminal">Terminal</a></li>
+                    </ul>
+                </div>
+            </div>
+            <hr>
+            <div class="social-media">
+                <ul class="social-icon">
+                    <li><a href="github"><img src="images/github.png" alt="github"></a></li>
+                    <li><a href="#linkedln"><img src="images/linkedln.png" alt="#linkedln"></a></li>
+                    <li><a href="#angel"><img src="images/angelist.png" alt="#angelist"></a></li>
+                    <li><a href="#twitter"><img src="images/twitter.png" alt="#twitter"></a></li>
+                    <li><a href="#medium"><img src="images/medium.png" alt="#medium"></a></li>
+                </ul>
+            </div>
+            <div class="work-icon"><img src="images/work.png" alt="#work-icon"></div>
+        </div>
+        <div class="works-header">
```

Comment:
- [ ] The container of the works section should be a `section` tag. Kindly have that fixed.